### ISSUE TITLE
Screen readers in virtual cursor mode can navigate to Announced component

### ISSUE THREAD

**Issue by davyrwoo** [microsoft/fluentui#12526]
### Environment Information

- **Package version(s)**: office-ui-fabric-react@7
- **Browser and OS versions**: Chrome 80, JAWS 2019.1912.1

### Describe the issue:

In virtual cursor mode, JAWS can navigate to the Announced component with up/down arrows

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/davyrwoo/pen/gOpJbQo?editable=true

#### Actual behavior:

In virtual cursor mode, JAWS can navigate to the Announced component with up/down arrows

#### Expected behavior:

Announced component should not be navigable even in virtual cursor mode.

**Comment by aneeshack4** [comment]
@davyrwoo Thanks for reaching out to us. Don't have access to JAWS but just tried this out with Narrator and I'm able to navigate with up/down arrows which is the correct behavior. I had to run it on the quick actions announced example on our site because your codepen didn't render. But ya either way this is by design and is working properly.
![image](https://user-images.githubusercontent.com/4161791/78415969-1abaee00-75da-11ea-956a-71af1c502b80.png)

**Comment by davyrwoo** [comment]
Hi @aneeshack4, 
In the codepen I linked, the component properly renders (it is just not visible) - the issue here is that the component is meant to be non-visible, but the AT can still navigate to them in virtual cursor/scan mode. I was able to repro it with Narrator as well. I think this might still be a valid issue with the Fluent component that can be looked into.

**Comment by aneeshack4** [comment]
@davyrwoo I don't understand - you're saying it should be "non-visible"? Why would this be invisible? Also scan mode on Narrator is a known buggy mode.

**Comment by davyrwoo** [comment]
@aneeshack4 The component isn't visibly rendering anything on the screen, so the screen reader should not be able to navigate to this component via keyboard navigation or scan mode. The screen reader should only be describing visible components, and it is not desired that it can navigate to content that is not visible on the screen.

**Comment by aneeshack4** [comment]
@davyrwoo Can you point to what about the Fluent component isn't following the ARIA spec? Because then this is actionable for us otherwise it isn't.
